Added more logging around the paths that lead to the creation of the query plan. (#203)
exasol/virtual-schema-common-document: doc/changes/changes_11.0.4.md

# Common Virtual Schema for Document Data 11.0.4, released 2025-06-24

> Code name: Add more logging around the paths that lead to the creation of the query plan.

Reviewer: ```suggestion
Code name: Improve query plan logging
```
Code name should be shorter :)

## Summary

> This release adds more logging around the paths that lead to the creation of the query plan.

Reviewer: Actually this just implements `toString()` for two classes...
```suggestion
This release implements `toString()` for classes `TableMapping` and `RemoteTableQuery`. This allows improving logging around the paths that lead to the creation of the query plan.
```

## Features

> * #202: Add more logging around the paths that lead to the creation of the query plan

Reviewer: ```suggestion
* #202: Implemented `toString()` for classes `TableMapping` and `RemoteTableQuery`
```
